docs/advanced/api.rst

```diff
@@ -30,12 +30,15 @@ For more details on configuring the firewall, see Fedora's `firewalld documentat
 On AWS, an external firewall is used instead, which will need to be configured similarly.
 
 To start the agents on a node, a configuration file must exist on the node at ``/etc/flocker/agent.yml``.
-This should be as follows, replacing ``${CONTROL_NODE}`` with the address of the control node:
+This should be as follows, replacing ``${CONTROL_NODE}`` with the address of the control node.
+The ``port`` variable is the port on the control node to connect to:
```

Comment:
Since 99% of the time it's going to be 4524, it would be nice if port was optional. Can be follow up issue since it's not critical, just nice to have.